libnetwork/driver_plugin.py

```diff
@@ -192,12 +358,23 @@ def join():
         remove_veth(host_interface_name)
         raise e
 
+    # TODO Optionally use the docker bridge, don't specify gateway
```

Comment:
Yup - this tied in with what you were doing - wanted to touch base with you on this first.